collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -83,6 +82,11 @@
     private AudioRecorderDependencyComponent audioRecorderDependencyComponent;
     private ProjectsDependencyComponent projectsDependencyComponent;
 
+    public Collect() {
+        super();
+        singleton = this;
```

Comment:
This needs to move here for anything that might need access to it before `onCreate`